### ISSUE TITLE
UNav footer NYS logo should have alt text or aria-label for screen readers (mirror header)

### ISSUE THREAD

**Issue by plasticmind** [ITS-HCD/nysds#290]
Should probably mirror the Unav header.

Currently:

![Image](https://github.com/user-attachments/assets/835bc5f1-44e8-40fa-ac2a-3b9f1ca31c6f)

In Header:

![Image](https://github.com/user-attachments/assets/98299943-dff6-4e20-8f97-ebb691bd2b00)

**Comment by NovaCat35** [comment]
PR fixed: https://github.com/ITS-HCD/nysds/pull/297
